src/Config.js

```diff
@@ -112,6 +112,9 @@ class ConfigOptions {
     approvalContext() { return "PR approval"; }
 
     copiedDescriptionSuffix() { return " (copied from PR by Anubis)"; }
+
+    // GitHub request timeout, ms
+    requestTimeout() { return 60000; }
```

Comment:
If this applies to _all_ bot communications with GitHub:

```suggestion
    // GitHub transaction timeout, ms
    // GitHub library default is zero which is interpreted as infinity (i.e. no timeout)
    githubTimeout() { return 60000; }
```